packages/web3-hooks/base/src/useReverseAddress.ts

```diff
@@ -5,13 +5,12 @@ import { useWeb3State } from './useWeb3State.js'
 export function useReverseAddress<T extends NetworkPluginID>(pluginID?: T, address?: string) {
     const { NameService } = useWeb3State(pluginID)
 
-    const isBad = !address || !NameService
-    return useQuery<string | undefined>({
+    return useQuery({
         queryKey: ['reverse', address],
-        enabled: !isBad,
+        enabled: !!NameService?.reverse,
         queryFn: async () => {
-            if (isBad) return
-            return NameService.reverse?.(address)
+            if (!address) return null
```

Comment:
Why not just return `undefined`? It seems cause unnecessary changes.

Reply:
react-query won't cache `undefined` result, it will try fetching again until non `undefined` result.

> It seems to cause unnecessary changes.

Those changes are typing only, no runtime change.